pkg/controller/builder_test.go

```diff
@@ -1,8 +1,10 @@
-package controller
+package controller_test
```

Comment:
What necessitated this change? Seems like it has the undesired side-effect of needing to export some types and functions that don't otherwise need to be exported.

Reply:
Tests _should_ be a separate package... if we don't want to do that, then I can put it back, but yes, that was a side effect.

Reply:
(reverted)